internal/services/v1/experimental.go

```diff
@@ -202,7 +211,194 @@ func (es *experimentalServer) BulkLoadRelationships(stream v1.ExperimentalServic
 		DispatchCount: 1,
 	})
 
-	return stream.SendAndClose(&v1.BulkLoadRelationshipsResponse{
+	return stream.SendAndClose(&v1.BulkImportRelationshipsResponse{
 		NumLoaded: numWritten,
 	})
 }
+
+func (es *experimentalServer) BulkExportRelationships(
+	req *v1.BulkExportRelationshipsRequest,
+	resp v1.ExperimentalService_BulkExportRelationshipsServer,
+) error {
+	ctx := resp.Context()
+	ds := datastoremw.MustFromContext(ctx)
+
+	var atRevision datastore.Revision
+	var cur dsoptions.Cursor
+
+	if req.OptionalCursor != nil {
+		var err error
+		atRevision, cur, err = decodeCursor(ds, req.OptionalCursor)
+		if err != nil {
+			return shared.RewriteError(ctx, err)
+		}
+	} else {
+		var err error
+		atRevision, _, err = consistency.RevisionFromContext(ctx)
```

Comment:
Interesting. That only just happens to work because I named the parameter the same as the other implementation. Is there any way to test that this happened? or enforce that the parameter is named appropriately?

Reply:
We should unit test it for each API method, but yeah: its based on naming, just like our consistency block

Reply:
I think I prefer the current implementation because it gets the revision and the cursor in a single decode.

Reply:
Okay, I guess the middleware will be a no-op